openedx/core/djangoapps/notifications/config/waffle.py

```diff
@@ -49,3 +49,12 @@
 # .. toggle_warning: When the flag is ON, Notifications Grouping feature is enabled.
 # .. toggle_tickets: INF-1472
 ENABLE_NOTIFICATION_GROUPING = CourseWaffleFlag(f'{WAFFLE_NAMESPACE}.enable_notification_grouping', __name__)
+
+# .. toggle_name: notifications.enable_push_notifications
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable push Notifications feature on mobile devices
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2025-05-13
+# .. toggle_warning: When the flag is ON, Notifications will go through ace push channels.
+ENABLE_PUSH_NOTIFICATIONS = CourseWaffleFlag(f'{WAFFLE_NAMESPACE}.enable_push_notifications', __name__)
```

Comment:
Is there a reason that we'd have this toggle in the long term?

If not, the please make the use cases `temporary` instead of `open_edx`, and add a target removal date that corresponds to when you'd expect to globally roll this out to all users.